### PR TITLE
Add docs for EXT_texture_norm16

### PR DESCRIPTION
**Pull request by Elchi3**
Hi! I'd like to contribute new documentation for this new WebGL extension.

Spec: https://www.khronos.org/registry/webgl/extensions/EXT_texture_norm16/
BCD: https://github.com/mdn/browser-compat-data/pull/7634

Question: This requires a new spec to be added for specData. Do I open a PR against https://github.com/mdn/yari/blob/master/kumascript/macros/SpecData.json or is it https://github.com/mdn/kumascript/blob/master/macros/SpecData.json?

Cheers and good luck for the launch today!

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Thanks @Elchi3 , I'll have a look at this soon.

To answer your question, the version of kumascript inside the yari repo is the one to update. The kumascript repo will be archived soon.

**Comment by Elchi3** [comment]
Thanks Chris, I will open a PR against Yari then.

**Comment by peterbe** [comment]
For the record, the repo is open for merges now.

File: files/en-us/web/api/ext_texture_norm16/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+slug: Web/API/EXT_texture_norm16
+tags:
+  - API
+  - Reference
+  - WebGL
+  - WebGL extension
+---
+<div>{{APIRef("WebGL")}}</div>
+
+<p>The <code><strong>EXT_texture_norm16</strong></code> extension is part of the <a href="/en-US/docs/Web/API/WebGL_API">WebGL API</a> and provides a set of new 16-bit signed normalized and unsigned normalized fixed point texture, renderbuffer and texture buffer formats.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ext_texture_norm16/index.html:12]:
```suggestion
<p>The <code><strong>EXT_texture_norm16</strong></code> extension is part of the <a href="/en-US/docs/Web/API/WebGL_API">WebGL API</a> and provides a set of new 16-bit signed normalized and unsigned normalized formats (fixed-point texture, renderbuffer and texture buffer).</p>
```
I realize this text is from https://www.khronos.org/registry/webgl/extensions/EXT_texture_norm16/ but its a pretty big pile of adjectives upon adjectives, so maybe for MDN we can un-pile it a bit…

File: files/en-us/web/api/ext_texture_norm16/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<p>When this extension is enabled:</p>
+<ul>
+  <li>The {{domxref("WebGLRenderingContext.texImage2D()")}} and {{domxref("WebGLRenderingContext.texSubImage2D()")}} methods accept new formats provided by this extension.</li>
+  <li>The 16-bit normalized fixed point types <code>ext.R16_EXT</code>, <code>ext.RG16_EXT</code> and <code>ext.RGBA16_EXT</code> become available as color-renderable formats and renderbuffers and be created in these formats.</li>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ext_texture_norm16/index.html:17]:
```suggestion
  <li>The 16-bit normalized fixed-point types <code>ext.R16_EXT</code>, <code>ext.RG16_EXT</code> and <code>ext.RGBA16_EXT</code> become available as color-renderable formats and renderbuffers and be created in these formats.</li>
```

File: files/en-us/web/api/ext_texture_norm16/index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<h2 id="Examples">Examples</h2>
+
+<h3>Enabling the extension</h3>
+<pre class="brush:js">
+let ext = gl.getExtension('EXT_texture_norm16');
+</pre>
+
+<h3>Texture formats</h3>
+
+<p>The {{domxref("WebGLRenderingContext.texImage2D()")}} accepts new formats when <code>EXT_texture_norm16</code> is enabled. Example calls:</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ext_texture_norm16/index.html:56]:
```suggestion
<p>The {{domxref("WebGLRenderingContext.texImage2D()")}} method accepts new formats when <code>EXT_texture_norm16</code> is enabled. Example calls:</p>
```

**Review by sideshowbarker — APPROVED** [review]
LGTM modulo one missing word and a few nits/suggestions

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
I set this to Draft status under the assumption that we don’t want it to end up getting merged until the SpecData change is merged and deployed. Did I assume correctly?

**Comment by Elchi3** [comment]
> I set this to Draft status under the assumption that we don’t want it to end up getting merged until the SpecData change is merged and deployed. Did I assume correctly?

The page builds without the macro change, so I'm not sure what the policy is. I've sent https://github.com/mdn/yari/pull/2147 now. If needed, someone can review and merge that one first.

**Comment by sideshowbarker** [comment]
> The page builds without the macro change, so I'm not sure what the policy is.

Yeah I guess we should have a documented policy. But at it least seems like if the change gets deployed to production with the new macro also having been deployed, the resulting user experience for readers of the article is no ideal — I mean, given that the spec reference would then just show up as *Unknown*, without the actual intended link to the spec.

**Comment by chrisdavidmills** [comment]
I guess the procedure in that case is just to make sure the correc spec details are there before the content pr is made. I'll write something up.

**Comment by Elchi3** [comment]
The Yari PR was merged. This is ready

**Comment by chrisdavidmills** [comment]
merged!

**Pull request merged**